lib/patches/active_model/dirty.rb

```diff
@@ -0,0 +1,3 @@
+ActiveModel::Dirty.class_eval do
+  alias_method :attributes_changed_by_globalize_setter, :attributes_changed_by_setter
```

Comment:
This is how ActiveModel deals with modifying `changed_attributes` internally, by aliasing to another method, so rather than accessing that private method I just did the same thing here.

See: https://github.com/shioyama/rails/blob/72f97e281059bc983eef5bc8915e53249c623dff/activemodel/lib/active_model/dirty.rb#L260